### PR TITLE
Hungry Serpent Snake Game in CPP

### PR DESCRIPTION
**Pull request by AnilKumarTeegala**

### CODE REVIEW

File: Hungry_Serpent/Hungry_Serpent.cpp
Comment (iamwatchdogs) [inline Hungry_Serpent/Hungry_Serpent.cpp:1]: Hey there @AnilKumarTeegala, You might be facing linting issues as your code is not following the coding standards of `clang`. If you need any help fixing code formatting, you can get help from the formatted version of your code at `show diffs and exit` step in the checks, you can find it here:

image

If you still facing issues, then please let us know.

**Review by iamwatchdogs — CHANGES_REQUESTED** [review]
Please update your code so that it can pass the lining checks. Please let us know if you need any help regarding any issue related to your PR.

File: Hungry_Serpent/Hungry_Serpent.cpp
Comment (AnilKumarTeegala) [reply Hungry_Serpent/Hungry_Serpent.cpp:1]: Updated the code and added a commit

**Review by iamwatchdogs — APPROVED** [review]
Review and Approved by @iamwatchdogs.

**Comment by iamwatchdogs** [comment]
Thank you for contributing @AnilKumarTeegala, make sure to check your contributions on [GitHub Pages](https://grow-with-open-source.github.io/C-CPP-Projects/).

**Pull request merged**